src/layman/layer/rest_layer_test.py

```diff
@@ -0,0 +1,41 @@
+import sys
+
+del sys.modules['layman']
+
+from layman import app, settings
+from test import flask_client
+from layman.util import url_for
+
+
+client = flask_client.client
+
+
+def test_sld_value(client):
+    username = 'test_layer_sld_user'
+    layername = 'test_layer_sld_layer'
+    style_url = f'http://{settings.LAYMAN_SERVER_NAME}/rest/{username}/layers/{layername}/style'
```

Comment:
Should be settings.LAYMAN_PROXY_SERVER_NAME, not LAYMAN_SERVER_NAME.

Reply:
Actually, url_for :)